ADR/ADR015-use-paper-trail-for-draft-live-or-some-other-solution.md

```diff
@@ -0,0 +1,57 @@
+# ADR015: Use Papertrail for draft/live or some other solution
+
+Date: 2023-03-03
+
+## Status
+
+Rejected
+
+## Context
+
+There is need to create live versions of a form. A form could potentially have multiple "live" forms during its lifetime
+but only one live form should be accessible at any given time. Using [PaperTrail](https://github.com/paper-trail-gem/paper_trail) 
+already creates an audit record of a form/pages record and stores the object in its previous state. 
+
+PaperTrail does have a 3rd party gem called [paper_trail-associations_tracking(PT-AT)](https://github.com/westonganger/paper_trail-association_tracking)
+that deals with [associations](https://github.com/paper-trail-gem/paper_trail#4b-associations).
+This gem should be able to give us the  functionality we needed such as trying to identify which versions of pages make 
+up a specific version of the form.
+
+### Positives
+- PT-AT can identify which forms/pages where created/updated and it was possible to restore that use to reuse
```

Comment:
Might just be a typo or me missing sopmething but I can't figure out what `it was possible to restore that use to reuse` means